src/ripple/app/paths/FindPaths.h

```diff
@@ -40,15 +41,15 @@ class FindPaths
         unsigned int const maxPaths);
     ~FindPaths();
 
-    bool findPathsForIssue (
+    boost::optional<std::pair<
+        STPathSet, std::vector<STAmount>>>
+    findPathsForIssue (
         Issue const& issue,
 
-        /** On input, pathsInOut contains any paths you want to ensure are
+        /** paths contains any paths you want to ensure are
             included if still good.
-
```

Comment:
This comment seems to have vanished, because that knowledge is now subsumed into that anonymous pair.  If you make it a struct, you could move this old comment to that struct...